javascript/index.js

```diff
@@ -14,52 +13,111 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  milUni.innerHTML = Number(milUni.innerHTML) + 1
+  printMilliseconds()
+  printMinutes()
+  printSeconds()
 }
 
 function printMinutes() {
-  // ... your code goes here
```

Comment:
the code inside your functions printMinutes, printSeconds and printMilliseconds work fine, but the idea was to work around the methods of the chronometer that are already created. You have access to the functions getMinutes, getSeconds, getMilliseconds and even the method that turns them into a string form with two digits (twoDigitsNumber). So by calling those from the chronometer object you could have got the information needed to update the DOM without needed to write all the logic again.